Translators/RoadConditions/README.md

```diff
@@ -0,0 +1,53 @@
+# Road Conditions Translator
+
+## Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Usage](#usage)
+- [Contributing](../CONTRIBUTING.md)
+
+## About <a name = "about"></a>
+
+Python Road Conditions to TIM message translator that is designed to pull messages from the CDOT Road Conditions feed and translate to TIM messages.
+
+## Getting Started <a name = "getting_started"></a>
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See [deployment](#deployment) for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+This project supports Python >= 3.5. Packages required to run the translator can be installed via [pip](https://pip.pypa.io/en/stable/)
+
+```bash
+pip install -r requirements.txt
+```
+
+Alternatively, if you are running VSCode there is a task available to run this on your behalf. This can be accessed under Task Explorer -> vscode -> pipinstall.
+
+The scripts also require access to the CDOT Postgres database. This can be accessed by setting the environment variables in the .env file. The .env file is not included in the repository for security reasons, however a sample.env file has been provided to show structure required. 
+
+In addition to the environment variables for accessing the CDOT Postgres database, the scripts also require the following environment variables to scrape the Road Conditions endpoint and deposit the resulting TIMs:
+<ol>
+    <li>CDOT_FEED_ENDPOINT - the CDOT Data Feed URL</li>
+    <li>CDOT_FEED_API_KEY - the API key for accessing the CDOT Road Conditions data</li>
+    <li>ODE_ENDPOINT - the ODE URL where translated TIMs will be submitted </li>

```

Comment:
(non-blocking) Since we'll be submitting to the TIM Manager instead of directly to the ODE, should we rename this to something like `TIM_MANAGER_ENDPOINT`?

Reply:
Changed!